server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/rfc/SerializerTest.scala

```diff
@@ -0,0 +1,177 @@
+/** **************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ * *
+ * http://www.apache.org/licenses/LICENSE-2.0                 *
+ * *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ * ***************************************************************/
+
+package scala.org.apache.james.jmap.rfc
+
+import java.net.URI
+
+import org.junit.jupiter.api.Test
```

Comment:
Can you have a look at Duc Viet work to use scala test runner?

Reply:
Do you mean all junit tests or just the serialization tests?

Reply:
I mean all. Let's try being consistent.